### DISCUSSION TITLE
company-select-next-or-abort quit when there is only company-tng-mode with only one candidates

### DISCUSSION THREAD

**Discussion by tshu-w** [company-mode/company-mode#1314] [Q&A] [answered]
Is it by designed? Should `company-select-next-or-abort` take `company-selection-default` into account like `company-select-next`?

## minimal config
```lisp
;;; emacs -Q -l ~/.config/emacs/test.el

(setq package-archives '(("melpa"        . "https://melpa.org/packages/")
                         ("org"          . "https://orgmode.org/elpa/")
                         ("gnu"          . "https://elpa.gnu.org/packages/")))
(package-initialize)

(package-install 'company)
(global-company-mode)
(company-tng-mode)
```

## step to reproduce
1. `emacs -Q -l ~/.config/emacs/test.el`
2. type `crea`
## 
<img width="614" alt="Screen Shot 2022-03-25 at 22 01 42" src="https://user-images.githubusercontent.com/13161779/160136737-2fdc92fb-93e1-462e-b595-eda3d1202765.png">
3. company quit when press `C-n`

**Comment by dgutov** [comment, marked as answer]
Hi!

Looks like a problem, yes. Should be fixed in latest master.

**Reply by dgutov** [reply]
If you have any further questions about this one, direct them to the issue.

**Reply by tshu-w** [reply]
hi, `company-select-previous-or-abort` also should be fixed.
